content/pages/volunteers.md

```diff
@@ -0,0 +1,96 @@
+Title: Volunteer Opportunities
+Category: pages
+Summary: Volunteer opportunities available with WiCS undergraduate.
+
+
+As we strive to improve the quality of our events, every contribution, large or
+small counts. Volunteering with our committee is a great way to get involved
+with us on a smaller scale. While being a committee member requires you to
+dedicate at least 8 hours per week to us, becoming a core volunteer is an
+easier, lower commitment way to be involved in our activities, help us run
+events and workshops, and get to know the committee members.  Volunteering is
+also a good way to figure out if you'd like to commit more time to WiCS and
+eventually apply to be a committee member.
+
+We have a range of volunteer positions, each of which will help you to learn
+about an aspect of our committee. By volunteering to be a **photographer**, you
+can learn about how we run events and help with publicizing them. By
+volunteering to be a **technical mentor** for our events, you can help teach
+technical concepts to others. By becoming a **wics-sys volunteer**, you can
+learn systems administration and static website development from our systems
+committee.
+
+## How volunteering works
+
+You can sign up for most positions by filling out
+[this](http://goo.gl/forms/NsOXea6Yjf5t02Oj1) form. The only exception is
```

Comment:
So the reasons I didn't use that form:
- we don't need volunteer activity descriptions on the form anymore because this page will have them. If we have them on the form as well, we'd have to maintain both the form and the page on the website
- we don't really need to ask people if they're specifically interested in photography or mentoring etc because we email the list as a whole with all the opportunities we have so far. (on the other hand in F15, we maintained separate lists of people who wanted to do, say, photography and emailed only them with photography opportunities which was pretty tedious).